crates/e2e/tests/e2e/protocol_fee.rs

```diff
@@ -88,46 +88,51 @@ async fn surplus_fee_sell_order_capped_test(web3: Web3) {
         max_volume_factor: 0.1,
     };
     // Without protocol fee:
-    // Expected executed_surplus_fee is 167058994203399
+    // Expected execution is 10000000000000000000 GNO for
```

Comment:
Why did those tests change?

Reply:
It's because of this code change: https://github.com/cowprotocol/services/pull/2448/files#diff-ceceb0ab17c45440702d1a3312aef5768ebad69027eac31792a581f9f486e0f4R159-R170

Here we skip adding the fee for sell orders. 
This is decided to be done with solver team to remove the special case (deduction by observed_fee) when the volume based cap is calculation in post processing: https://github.com/cowprotocol/solver-rewards/blob/main/queries/orderbook/batch_rewards.sql#L89

Reply:
Btw we have an ongoing discussion again how to define trade volume more robustly and clearly.